vpplink/vmxnet3.go

```diff
@@ -0,0 +1,94 @@
+// Copyright (C) 2020 Cisco Systems Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or
+// implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package vpplink
+
+import (
+	"encoding/binary"
+	"fmt"
+	"regexp"
+	"strconv"
+
+	"github.com/pkg/errors"
+	"github.com/projectcalico/vpp-dataplane/vpplink/binapi/vppapi/vmxnet3"
+)
+
+type Vmxnet3Info struct {
```

Comment:
It would be a bit more consistent to declare a Vmxnet3Interface struct in the `types` package.